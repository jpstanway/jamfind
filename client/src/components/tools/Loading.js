import React from "react";
import loading from "../../imgs/loading.gif";

const Loading = () => {
  return (
    <div className="container">
      <div className="row">
        <div className="col-md-12 text-center">
          <img src={loading} />
        </div>
      </div>
    </div>
  );
};

export default Loading;
