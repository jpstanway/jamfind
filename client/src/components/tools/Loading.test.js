import React from "react";
import { shallow } from "enzyme";
import Loading from "./Loading";

it("renders loading component", () => {
  expect(shallow(<Loading />)).toMatchSnapshot();
});
