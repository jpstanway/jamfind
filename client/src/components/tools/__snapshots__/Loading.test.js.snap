// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders loading component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loading />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="row"
      >
        <div
          className="col-md-12 text-center"
        >
          <img
            alt="Loading..."
            src="https://res.cloudinary.com/mtninja/image/upload/v1551554469/loading_yunyf4.gif"
          />
        </div>
      </div>,
      "className": "container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="col-md-12 text-center"
        >
          <img
            alt="Loading..."
            src="https://res.cloudinary.com/mtninja/image/upload/v1551554469/loading_yunyf4.gif"
          />
        </div>,
        "className": "row",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <img
            alt="Loading..."
            src="https://res.cloudinary.com/mtninja/image/upload/v1551554469/loading_yunyf4.gif"
          />,
          "className": "col-md-12 text-center",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "Loading...",
            "src": "https://res.cloudinary.com/mtninja/image/upload/v1551554469/loading_yunyf4.gif",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="row"
        >
          <div
            className="col-md-12 text-center"
          >
            <img
              alt="Loading..."
              src="https://res.cloudinary.com/mtninja/image/upload/v1551554469/loading_yunyf4.gif"
            />
          </div>
        </div>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="col-md-12 text-center"
          >
            <img
              alt="Loading..."
              src="https://res.cloudinary.com/mtninja/image/upload/v1551554469/loading_yunyf4.gif"
            />
          </div>,
          "className": "row",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              alt="Loading..."
              src="https://res.cloudinary.com/mtninja/image/upload/v1551554469/loading_yunyf4.gif"
            />,
            "className": "col-md-12 text-center",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "Loading...",
              "src": "https://res.cloudinary.com/mtninja/image/upload/v1551554469/loading_yunyf4.gif",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
